src/intercooler.js

```diff
@@ -1732,8 +1734,11 @@ var Intercooler = Intercooler || (function() {
         if (historyData) {
           processICResponse(historyData["content"], getTargetForHistory($('body')), true);
           if (historyData["yOffset"]) {
-            window.scrollTo(0, historyData["yOffset"])
-          }
+            window.scrollTo(0, historyData["yOffset"]);
+           }
```

Comment:
Looks like this and the next few lines are unintentionally indented by one extra space.

Reply:
Fixed.